packages/frontEnd/src/views/categoryDetails/pagination/Pagination.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+interface Props {
+  totalPages: number;
+  changeCurrentPage: (page: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  totalPages,
+  changeCurrentPage,
+}) => {
+  const mapPages = () => {
+    return Array.from(Array(totalPages).keys()).map((page) => {
+      return (
+        <div
+          className="bg-dark-blue p-4 text-white ml-4"
+          key={page}
+          onClick={() => changeCurrentPage(page)}
+        >
+          {page}
+        </div>
+      );
+    });
+  };
+
+  return (
```

Comment:
I don't know exactly how the filtration works  did you do any test?

Reply:
It doesn't work because doesn't exist